docs/notes/feature-replace_in_fields.md

```diff
@@ -0,0 +1,17 @@
+# Replacing text in fields
+
+There is a new form of the 'replace' command which works directly
+with fields:
+
+  replace <pattern> with <text> in <field chunk> (replacing | preserving) styles
```

Comment:
To make this formatted as code you'll need to indent it with either a tab or at least 4 spaces.